content/800-data-platform/200-projects/130-import-existing-project-new-db.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Import an existing project with a new database'
+metaTitle: 'Import an existing project with a new database in the Prisma Data Platform'
+metaDescription: 'If you want to test how your Node.js application is created, managed, and runs in the Prisma Data Platform, you can import your repository and connect the application to a new database. The Prisma Data Platform applies all schema migrations to the new database.'
+tocDepth: 3
+---
+
+<TopBlock>
+
+Import your application and use an empty database to explore and test project creation and the features in the Prisma Data Platform.
+
+You can use the steps provided in [Import an existing project with an existing database](/data-platform/create-and-manage-projects/import-existing-project-existing-db) to import a project.
+
+At step 4.1., in which you provide the database connection string, make sure to provide a connection string to an empty database.
```

Comment:
I think we should lose the dot after "4.1".

Also, I think "ensure that you" or "make sure that you" would be more natural than "make sure to". Or, to keep it really brief, just "... string, provide a connection string..."